src/nfd/nfd-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-nfd
```

Comment:
Please merge yaml files together where it makes sense. Similar concept found here: https://github.com/Azure/maximo/blob/main/src/mas/mas-operator.yaml

Reply:
Files merged, and installer script updated to reflect new files:

https://github.com/Azure/maximo/pull/15/commits/5c84134801f4ea715f2c51ab83e3b7ade74cfc38